src/main/java/com/synopsys/integration/alert/Application.java

```diff
@@ -114,7 +116,9 @@ public SimpleJobLauncher jobLauncher() {
 
     @Bean
     public TaskScheduler taskScheduler() {
-        return new ThreadPoolTaskScheduler();
+        final ThreadPoolTaskScheduler threadPoolTaskScheduler = new ThreadPoolTaskScheduler();
+        threadPoolTaskScheduler.setPoolSize(Runtime.getRuntime().availableProcessors());
```

Comment:
Is this restricted by the security manager? If so, we should do some sort of check and set it to a default otherwise.